config/environments/production.rb

```diff
@@ -19,7 +19,8 @@
   config.public_file_server.enabled = ENV['RAILS_SERVE_STATIC_FILES'].present?
 
   # Compress JavaScripts and CSS.
-  config.assets.js_compressor = :uglifier
+  config.assets.js_compressor = Uglifier.new(harmony: true) # ES6 support
```

Comment:
@alexsoble @jhilde This looks like the problem - no bueno for IE11.  Can we just revert this PR to bring the site back up?

Reply:
👍